### DISCUSSION TITLE
is it possible to search specific campground in Yellowstone?

### DISCUSSION THREAD

**Discussion by zwyolo** [juftin/camply#15] [Q&A] [answered]
Hi, camply is such a great tool to book a campground. however just wondering if we can search specific campground in Yellowstone? Yellowstone is so big, if we can stay in different campgrounds, it will be more convenient for us to arrange our tour. Thx!

**Comment by juftin** [comment, marked as answer]
## Update

Yes! Here's how it works: Yellowstone is kinda goofy, most of the campgrounds in the park are hosted by [yellowstonenationalparklodges.com](https://www.yellowstonenationalparklodges.com/stay/camping/). To search these campgrounds you will set the `--provider` parameter to `yellowstone`.  Currently, there are 5 supported campgrounds, and you can specify which campgrounds you want searched by passing the `--campground` parameter:

Eligible Campgrounds (`camply` campground ID):
1) Canyon Campground (`YLYC:RV`)
2) Bridge Bay Campground (`YLYB:RV`)
3) Grant Campground (`YLYG:RV`)
4) Madison Campground (`YLYM:RV`)
5) Fishing Bridge RV Park (`YLYF:RV`)

```
camply campsites \
    --provider yellowstone \
    --campground YLYC:RV \
    --campground YLYB:RV \
    --campground YLYG:RV \
    --start-date 2021-07-25 \
    --end-date 2021-07-31
```

Separately, there are currently 3 campgrounds inside of Yellowstone that are hosted by [recreation.gov](https://www.recreation.gov/camping/gateways/2988) instead of  [yellowstonenationalparklodges.com](https://www.yellowstonenationalparklodges.com/stay/camping/) (Recreation Area ID # `2988`):
1) Pebble Creek Campground (`259307`)
2) Mammoth Campground (`247571`)
3) Slough Creek Campground (`259310`)

Here's how you would search for a select few of those campsites:
```
camply campsites \
    --campground 259307 \
    --campground 247571 \
    --campground 259310 \
    --start-date 2021-07-25 \
    --end-date 2021-07-31
```

![image](https://user-images.githubusercontent.com/49741340/120429070-05526880-c332-11eb-8655-6c1b2a5aba84.png)

**Reply by zwyolo** [reply]
Thanks for the quick reply! Thanks to the CAMPLY, I have successfully booked campground in Yellowstone for the travel on independence day! Hope to see more functions come to the CAMPLY!

**Reply by juftin** [reply]
Great news, so glad camply was helpful for you. Have a great 4th of July!
